docs/hardware-interfaces/sdio.md

```diff
@@ -0,0 +1,67 @@
+---
+title: SDIO / eMMC
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# SDIO / eMMC
+
+## Introduction
+The Omega2 features an SDIO/eMMC interface that enables the use of SD/MicroSD cards or eMMC storage chips.
+
+**Omega2:** Does not have a microSD slot and does not expose the SDIO/eMMC interface.
+
+**Omega2+:** Has a microSD slot.
+
+**Omega2S/2S+:** Provides the SDIO/eMMC interface on exposed pins.
+
+## Hardware
+The Omega2 has a single SDIO interface.
+
+Only one device can be specified on the interface port at design time: eMMC, SD/Micro-SD, or other SDIO.
+
+The interface supports the SDXC specification for SD cards, with a maximum capacity of 2 TB and a maximum transfer speed of 300 MB/s. It also supports the eMMC5.1 interface for eMMC storage.
+
+<Tabs>
+  <TabItem value="omega2" label="Omega2" default>
+  
+  The microSD slot is highlighted on the Omega2+ photo below.
+
+
+![omega2-microSD-photo](./assets/omega2-microSD-photo.jpg)
+
+  </TabItem>
+  <TabItem value="omega2s" label="Omega2S">
+  
+  The SDIO/eMMC pins are highlighted on the Omega2S diagram below.
+  
+
+![omega2s-sdio_emmc-diagram](./assets/omega2s-sdio_emmc-pinout.png)
+
+  </TabItem>
+</Tabs>
+
+See [Omega2S hardware design guide](https://github.com/OnionIoT/Omega2/blob/master/Documents/Omega2S%20Hardware%20Design%20Guide.pdf) for more information and recommendations on designing boards with microSD card slots or eMMC.
```

Comment:
What do you think about folding this into the Omega2S tab above? 
It only applies to the omega2s/2s+, boards that use the Omega2+ must use the microSD slot on the Omega2+ itself.